### PR TITLE
🌐 Add Chinese translation for Advanced - Path Operation Advanced Configuration

### PR DESCRIPTION
**Pull request by RunningIkkyu**
Add Chinese docs for advanced/path-operation-advanced-configuration.md. Could you help review plz. @waynerv  @Dustyposa

### CODE REVIEW

File: docs/zh/docs/advanced/path-operation-advanced-configuration.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# 路径操作的高级配置
+
+## OpenAPI 的 operationId
+
+!!! warning
+    如果你并非 OpenAPI 的「专家」，你可能不需要这部分内容。
+
+你可以通过参数 `operation_id` 将 OpenAPI 的 `operationId` 设置为在路径操作中使用。
```
Comment (waynerv) [inline docs/zh/docs/advanced/path-operation-advanced-configuration.md:8]: 你可以在路径操作中通过参数 `operation_id` 设置要使用的 OpenAPI `operationId`。

Comment (waynerv) [reply docs/zh/docs/advanced/path-operation-advanced-configuration.md:8]: The segmentation should be "You can set the OpenAPI operationId to be used | in your path operation | with the parameter operation_id."

File: docs/zh/docs/advanced/path-operation-advanced-configuration.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+务必确保每次操作的 `operation_id` 都是唯一的。
```
Comment (waynerv) [inline docs/zh/docs/advanced/path-operation-advanced-configuration.md:10]: "each operation" could translate to  "每个路径操作“

File: docs/zh/docs/advanced/path-operation-advanced-configuration.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+```Python hl_lines="6"
+{!../../../docs_src/path_operation_advanced_configuration/tutorial001.py!}
+```
+
+### 使用 *路径操作函数* 的函数名作为 operationId
+
+如果你想用你的 API 的函数名作为 `operationId` 的名字，你可以遍历一遍 API 的函数名，然后使用他们的 `APIRoute.name` 重写每个 *路径操作* 的 `operation_id`。
+
+在此之前你要给所有函数添加 *路径操作*。
```
Comment (waynerv) [inline docs/zh/docs/advanced/path-operation-advanced-configuration.md:20]: 你应该在添加了所有路径操作之后执行此操作。

File: docs/zh/docs/advanced/path-operation-advanced-configuration.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+```Python hl_lines="2 12 13 14 15 16 17 18 19 20 21 24"
+{!../../../docs_src/path_operation_advanced_configuration/tutorial002.py!}
+```
+
+!!! tip
+    如果你手动调动 `app.openapi()`，你应该在此之前更新 `operationId`。
```
Comment (waynerv) [inline docs/zh/docs/advanced/path-operation-advanced-configuration.md:27]: ”call“ always translate to "调用".Is here a typo?

**Review by waynerv — CHANGES_REQUESTED** [review]

**Comment by RunningIkkyu** [comment]
@waynerv  Thanks for your review.

**Review by waynerv — APPROVED** [review]

**Comment by RunningIkkyu** [comment]
@Dustyposa @phy25 @yanever @Laineyzhang55   Please take a look.

**Review by Laineyzhang55 — APPROVED** [review]

**Comment by tiangolo** [comment]
Thank you @RunningIkkyu ! :clap: :clap: :cake: 

And thanks @waynerv and @Laineyzhang55 for the reviews :bowing_man: :coffee:

**Pull request merged**